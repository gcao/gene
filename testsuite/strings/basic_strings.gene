#!/usr/bin/env gene run
# TEST: Basic string operations

# String literals
(print "Hello")
(print "World")

# String interpolation
(var name "Gene")
(var version 1)
(print #"Welcome to {name} v{version}!")