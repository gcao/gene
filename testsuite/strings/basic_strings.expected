Hello
World
Welcome to Gene v1!